agoricco/__init__.py

```diff
@@ -5,16 +5,20 @@
 from flask_cors import CORS
 from flask_seasurf import SeaSurf
 from flask_mobility import Mobility
+from flask_babel import Babel
 
 
 app = Flask(__name__)
+babel = Babel(app)
 
 
 if 'FLASK_APP_SETTINGS' in os.environ:
     app.config.from_envvar('FLASK_APP_SETTINGS')
 else:
     app.config.from_object('agoricco.default_settings')
 
+app.config['BABEL_DEFAULT_LOCALE'] = 'fr'
```

Comment:
Does this need to be set? Or does `BABEL_DEFAULT_LOCALE` get a default from babel? If it must be set, I would ask that it goes in the `default_settings.py` file.